pkg/validations/upgradevalidations/cluster.go

```diff
@@ -28,8 +28,21 @@ func ValidateClusterObjectExists(ctx context.Context, k validations.KubectlClien
 
 func ValidateTaintsSupport(ctx context.Context, clusterSpec *cluster.Spec) error {
```

Comment:
Is this only an upgrade validation?

Reply:
No, but this is the same method and being reused for create as well.

Reply:
I think another comment mentioned it, but can we move this validation to a shared package then instead of under upgrades?

Reply:
this was moved to a common package, `pkg/validations/input`